.golangci.yaml

```diff
@@ -0,0 +1,11 @@
+linters:
+  # Disable all linters.
+  # Default: false
+  disable-all: true
```

Comment:
do we lose coverage?

we were running the linters before AFAIK as part of https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/cloud-provider-gcp/755/cloud-provider-gcp-verify-all/1828578606167625728